health/micro-ui-react19/web/packages/css/src/pages/employee/commodityManagement.scss

```
@@ -0,0 +1,492 @@
+/* ===================================================================
+   Commodity Management Module - Styles
+   =================================================================== */
+
+/* --- Dashboard Layout --- */
+
+.cm-header {
+  margin: 0 0 1.5rem 0;
+  font-family: var(--digitv2-fontFamily-rc);
+  font-weight: var(--digitv2-fontWeight-bold);
+  font-style: Bold;
```

Stylelint (17.6.0)

[error] Expected "Bold" to be "bold" (value-keyword-case)

(value-keyword-case)